decentraland/screens/EthereumSignUpScreen.js

```diff
@@ -39,7 +66,7 @@ export class EthereumSignUpScreen extends React.Component {
   _onSignUp = () => {
     const { setSetupInProgress } = this.props;
     setSetupInProgress(true);
-    showInfo(`Creating and funding account...`);
```

Comment:
Is there no equivalent for this one anymore?

Reply:
I've removed it because the toast now is being shown when some action was completed and the text on the screen is being used to show when something is processing.

Reply:
Ah ok, so for this one it would just be "buy" clicked, and that would be silly? Got it.

Reply:
Just after clicking the button "Continue" the user will back to "BuyLand" screen and the message "Generating account..." will be shown.

Reply:
Okay, sounds good! Resolving this.